internal/flink/command.go

```diff
@@ -22,6 +23,10 @@ func New(cfg *config.Config, prerunner pcmd.PreRunner) *cobra.Command {
 
 	c := &command{pcmd.NewAuthenticatedCLICommand(cmd, prerunner)}
 
+	if cfg.IsTest || featureflags.Manager.BoolVariation("cli.flink.connection", cfg.Context(), config.CliLaunchDarklyClient, true, false) {
```

Comment:
Are we going to remove the `cfg.IsTest` part when the developing is finished and fully relying on the LD flag?

Reply:
I think we keep it. It's for us, to make sure that our tests/pipeline is always able to run these commands.